pvp-faq.md

```diff
@@ -230,6 +230,23 @@ executables such as `alex` or `happy` which generate code, and
 especially when such packages are meant to be tracked as dependencies
 in `build-tools`.
 
+### My Package provides multiple APIs; how does the PVP apply to multi-library packages?
+
+Packages that expose multiple public libraries must consider their API
```

Comment:
I suggest putting not only Cabal version but also GHC version: [as one of confused users learned today](https://matrix.to/#/!VeTpfwcnSnUZwjyccf:libera.chat/$6npgZbLqfU6ewVhWBT7B3NJg0X9gxPnpTAMmSE9VnPE?via=libera.chat&via=matrix.org&via=mozilla.org), the feature is only supported starting with GHC 8.8.

Reply:
Since this is a FAQ, I think the primary audience is probably "people who have realised that they want to use multiple public libraries but don't know what the PVP recommends", so they should likely know the feature. But I added some pointers anyway, can't hurt.